docs/metrics.md

```diff
@@ -75,6 +75,183 @@ The docker image exposes StatsD via the (standard) port 8125 and a Grafana dashb
+
+##### Container Run
+
+* `openwhisk.counter.invoker_activationRun_start` (counter) - Count of action executions performed.
+* `openwhisk.histogram.invoker_activationRun_finish` (histogram) - Time taken for action execution for success case.
+* `openwhisk.histogram.invoker_activationRun_error` (histogram) - Time taken for action execution for failed cases. Count metrics of this histogram would give insight on failed execution count.
+
+##### Container Start
+
+* `openwhisk.counter.invoker_containerStart.cold_count` (counter) - Count of number of cold starts.
+* `openwhisk.counter.invoker_containerStart.recreated_count` (counter) - Count of number of times container is recreated.
+* `openwhisk.counter.invoker_containerStart.warm_count` (counter) - Count of number of times a warm container is used.
+
+##### Log Collection
+
+* `openwhisk.counter.invoker_collectLogs_start` (counter) - Count of number of times log were collected.
+* `openwhisk.counter.invoker_collectLogs_error` (counter) - Count of number of failed logs collections.
+* `openwhisk.histogram.invoker_collectLogs_error` (histogram) - Time taken for failed log collection.
+* `openwhisk.histogram.invoker_collectLogs_finish` (histogram) - Time taken for successful log collection.
+
+##### Activation Handling
+
+* `openwhisk.counter.invoker_activation_start` (counter) - Count of activations handled
+
+##### Docker Metrics
+
+Following metrics capture stats around various docker command executions.
+
+* Pause
+  * `openwhisk.counter.invoker_docker.pause_start`
+  * `openwhisk.counter.invoker_docker.pause_error`
+  * `openwhisk.histogram.invoker_docker.pause_finish`
+  * `openwhisk.histogram.invoker_docker.pause_error`
+* Ps
+  * `openwhisk.counter.invoker_docker.ps_start`
+  * `openwhisk.counter.invoker_docker.ps_error`
+  * `openwhisk.histogram.invoker_docker.ps_finish`
+  * `openwhisk.histogram.invoker_docker.ps_error`
+* pull
+  * `openwhisk.counter.invoker_docker.pull_start`
+  * `openwhisk.counter.invoker_docker.pull_error`
+  * `openwhisk.histogram.invoker_docker.pull_finish`
+  * `openwhisk.histogram.invoker_docker.pull_error`
+* rm
+  * `openwhisk.counter.invoker_docker.rm_start`
+  * `openwhisk.counter.invoker_docker.rm_error`
+  * `openwhisk.histogram.invoker_docker.rm_finish`
+  * `openwhisk.histogram.invoker_docker.rm_error`
+* run
+  * `openwhisk.counter.invoker_docker.run_start`
+  * `openwhisk.counter.invoker_docker.run_error`
+  * `openwhisk.histogram.invoker_docker.run_finish`
+  * `openwhisk.histogram.invoker_docker.run_error`
+* unpause
+  * `openwhisk.counter.invoker_docker.unpause_start`
+  * `openwhisk.counter.invoker_docker.unpause_error`
+  * `openwhisk.histogram.invoker_docker.unpause_finish`
+  * `openwhisk.histogram.invoker_docker.unpause_error`
+
+#### Kafka Metrics
+
+Metrics below are emitted per kafka topic.
+
+* `openwhisk.histogram.kafka_<topic name>.delay_start` - Time delay between when a message was pushed to kafka and when it is read within a consumer.
```

Comment:
should we also mention that Delay is being emitted for each pool by Invoker, while Queue metric is emitted every 10 seconds?

Reply:
Missed that it was be done via a scheduled task. Per default config it should be emitted every 60 seconds